boards/waspmote-pro/Kconfig

```diff
@@ -11,11 +11,12 @@ config BOARD_WASPMOTE_PRO
     bool
     default y
     select CPU_MODEL_ATMEGA1281
+    select HAS_ARDUINO
     select HAS_PERIPH_ADC
     select HAS_PERIPH_I2C
     select HAS_PERIPH_SPI
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
-    select HAS_ARDUINO
+    select HAS_SDCARD_SPI
```

Comment:
You also want to add

    FEATURES_PROVIDED += sdcard_spi

to `Makefile.features`

Reply:
done